### ISSUE TITLE
Proposal: Sprint: Packaging and Publishing in Python

### ISSUE THREAD

**Issue by brownsarahm** [carpentrycon/carpentryconhome-proposals#21]
### Session details

* Session type:  resource development sprint

* Keywords: lesson development, python, packaging, publishing

* Permission to record this session: No

### Abstract

This 48 hour sprint will follow a CarpentryCon at Home run of the workshop [Packaging and Publishing in Python](https://carpentries-incubator.github.io/python-packaging-publishing/). The sprint will take place in the lesson's [Carpentries-Incubator repository](https://github.com/carpentries-incubator/python-packaging-publishing). There will be issues added prior to and during the pilot run and a roadmap for the direction of the lessons in the repository's projects for full transparency of requested and prioritized contributions. 

You should participate in this if:
- You attend the workshop (#20) and have ideas for expansion.
- You want to learn about the topic but have conflict with the session.
- You are interested in learning about lesson maintenance or the Carpentries Incubator by trying it out.

### Personal details

* Name or pseudoname of the session lead: Sarah Brown

* Co-leads' names (we recommend involving 2 helpers/co-leads): N/A? no synchronous component but if people are interested in maintaining then they're welcome. 

* Email or other ways to contact the session leads/co-leads: smb@sarahmbrown.org

* Country of residence and/or compatible Time Zones (provide options): N/A

* Would you like to present this multiple times, in other time zones: No 

* Would you like to volunteer to be listed as a wrangler/host for your time zone: No  

### Is there any help you would like to invite from the community? Please provide below in bullet points.

* inviting co-maintainers as helpers, [application available as issue template](https://github.com/carpentries-incubator/python-packaging-publishing/issues/new/choose) on the repository
* looking for issue-documenters during the workshop (see: #20 ) to add issues for the sprint

* Other comment: WRITE HERE

**Comment by ChristinaLK** [comment]
@brownsarahm I'm going to tag this as "help wanted" as it sounds like you're still looking for help. We're hoping to point the community at this repository next week - can you reiterate your asks for help in a comment so that it's visible at the top and (current) bottom of the issue?

**Comment by serahkiburu** [comment]
Hi @brownsarahm, the organising committee reviewed this session proposal and had this to say:

>I am so excited to see this happen, it will be invaluable for the emerging community in the Carpentries Incubator program. Since some details are missing such as what the discussion points and expected outcome of this session are, we request the session lead to include those details in their abstract before the program is publicly announced.

As I mentioned in #20, we will be sharing more details i.e. around scheduling with you in the coming days. Let me know if I can answer any questions or clarify anything for the time being.

**Comment by kerchner** [comment]
Hi @brownsarahm .  Am I understanding correctly that this (#21) isn't an event per se, but more of a 48-hour period that will roughly follow the sessions of #20?  I can schedule you for two sessions of #20 to help increase the number of people prepared to participate in the sprint.

**Comment by brownsarahm** [comment]
Yes, this should be after #20, but need not follow immediately. A little time in between to structure the requests/ add my own issues post teaching will be helpful.

**Comment by kerchner** [comment]
If #20 will be occurring on Tues/Thurs July 21/23, then when would you like to have the sprint be?  Perhaps Monday/Tuesday of the following week?  Up to you - but we can help put the sprint on the schedule so participants can plan for it.

**Comment by brownsarahm** [comment]
that works.
